src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -88,6 +88,10 @@ public void setResourceIdResolver(ResourceIdResolver resourceIdResolver) {
 		this.simpleMessageListenerContainerFactory.setResourceIdResolver(resourceIdResolver);
 	}
 
+	public void setAutoStartUp(boolean autoStartUp) {
+		this.simpleMessageListenerContainerFactory.setAutoStartup(autoStartUp);
+	}
```

Comment:
You have to override `setAutoStartup(boolean autoStartup)` method instead, do this setting and invoke `super`.
Exactly that `auto-startup="false"` invoke `setAutoStartup()` method. And yes, only after that change your test will be fine.